generators/server/templates/gradle.properties.ejs

```diff
@@ -21,7 +21,7 @@ profile=dev
 
 # Build properties
 nodeVersion=<%= NODE_VERSION %>
-npmVersion=<%= NPM_VERSION %>
+npmVersionProperty=<%= NPM_VERSION %>
```

Comment:
Why the change in name? Just curious.

Reply:
Somehow the [property conflicts with the property name of the plugin](https://github.com/jhipster/generator-jhipster/issues/13746). As it is already the case for the old release version I suspect some gradle update or it never worked as expected (as we usually use npm directly instead maven/gradle in our ci).

So we could change the name or inline the versions in `build.gradle` as the properties are not used somewhere else.